spec/models/miq_request_workflow_spec.rb

```diff
@@ -1,10 +1,14 @@
 require "spec_helper"
 
 describe MiqRequestWorkflow do
-  let(:workflow) { FactoryGirl.build(:miq_provision_workflow) }
+  let(:requester) { FactoryGirl.create(:user_admin) }
+  let(:dialog) { FactoryGirl.create(:miq_dialog_provision) }
+  let(:workflow) do
+    MiqProvisionWorkflow.new({:provision_dialog_name => dialog.name}, requester.userid)
+  end
```

Comment:
All you need to add is `let(:requester) { workflow.instance_variable_get(:@requester) }`